AccessCheckoutSDK/AccessCheckoutSDKTests/view/PanTextChangeHandlerTests.swift

```diff
@@ -0,0 +1,107 @@
+@testable import AccessCheckoutSDK
+import XCTest
+
+class PanTextChangeHandlerTests : XCTestCase {
```

Comment:
Does it also support deleting a space?

Reply:
I have added a test that confirms it returns the same string when deleting a space, the behaviour is that the cursor moves across the space but the formatting is maintained

Reply:
I have also a test for the case where you delete spaces and characters

Reply:
Awesome